app/src/main/res/menu/bottom_navigation_menu.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu xmlns:android="http://schemas.android.com/apk/res/android">
+  <item
+      android:id="@+id/page_1"
```

Comment:
Didn't see where this is being used exactly. It looked like some list, but maybe its for the nav bar. Could you raname these to more exact names like `page_recipes`?

Reply:
Yeah, this is where the items in the navigation bar are defined. I'll rename them to be more clear.